packages/next/tsconfig.json

```diff
@@ -2,13 +2,14 @@
   "extends": "../../tsconfig-tsec.json",
   "compilerOptions": {
     "strict": true,
-    "module": "esnext",
-    "target": "ES2017",
+    "stripInternal": true,
     "esModuleInterop": true,
-    "moduleResolution": "node",
+    "skipDefaultLibCheck": true,
+    "verbatimModuleSyntax": true,
     "jsx": "react-jsx",
-    "stripInternal": true,
-    "verbatimModuleSyntax": true
+    "module": "ESNext",
+    "target": "ES2018",
```

Comment:
Basically no change but `target ES2017 -> ES2018`, sorry for moving around order. 😅

### Why?

![why_tsconfig_target_es2018](https://github.com/vercel/next.js/assets/120007119/c3ea58e8-d31d-4aaf-9d2a-9c6110091b66)